android/gradle.properties

```diff
@@ -1,5 +1,5 @@
-PosthogReactNativeSessionReplay_kotlinVersion=1.7.0
+PosthogReactNativeSessionReplay_kotlinVersion=1.8.0
 PosthogReactNativeSessionReplay_minSdkVersion=21
-PosthogReactNativeSessionReplay_targetSdkVersion=31
-PosthogReactNativeSessionReplay_compileSdkVersion=31
-PosthogReactNativeSessionReplay_ndkversion=21.4.7075529
+PosthogReactNativeSessionReplay_targetSdkVersion=34
+PosthogReactNativeSessionReplay_compileSdkVersion=34
```

Comment:
I remember messing with sdk versions was always messy given how fragmented Android was as an ecosystem. I assume the target / compile versions are less important than changing something like the `minSdkVersion`?

Reply:
they are equally important, if you dont compile and target the newest versions, your app can't even ben released in the stores.
this is just a fallback though, in case its not provided by the "final app"